release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2752,4 +2752,9 @@ You can view the container images in this release by running the following comma
 ----
 $ oc adm release info 4.17.0 --pullspecs
 ----
+
+[id="ocp-4-17-0-updating_{context}"]
+==== Updating
+To update an {product-title} 4.17 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Comment:
Does this make sense to say 4.17 when this is the first possible version of 4.17? What's the established practice here for .0 z stream versions?

Reply:
I updated it to 4.16 now, so it it more accurate. My apologies.